### PR TITLE
ansible-scylla-node: Use Scylla API for getting existent tokens instead of nodetool ring

### PR DESCRIPTION
**Pull request by igorribeiroduarte**

### CODE REVIEW

File: ansible-scylla-node/tasks/generate_tokens.yml [hunk 1]
```diff
@@ -16,38 +16,48 @@
...
 
-- name: Prepare old tokens list
-  set_fact:
-    token_list_str: "{{ token_list.keys() | zip(token_list.values()) | map('join', '=') | join('\n') | replace('[','') | replace(']','') | replace(\"'\", '') | replace(' ', '') }}"
-  when: token_list is defined
+- name: Create an empty tokens file
+  copy:
+    content: ""
+    dest: /tmp/tokens_file
+  delegate_to: localhost
```
Comment (vladzcloudius) [inline ansible-scylla-node/tasks/generate_tokens.yml:30]: This won't work well when multiple concurrent instances of the job would be running.
Generate a temp file instead, please.

File: ansible-scylla-node/tasks/generate_tokens.yml [hunk 2]
```diff
@@ -16,38 +16,48 @@
...
+
+- name: Get existent tokens
```
Comment (vladzcloudius) [inline ansible-scylla-node/tasks/generate_tokens.yml:32]: spelling: s/existent/existing/

File: ansible-scylla-node/tasks/generate_tokens.yml [hunk 3]
```diff
@@ -16,38 +16,48 @@
...
+  block:
+    - name: Get tokens
+      uri:
+        url: "http://{{ scylla_api_address }}:{{ scylla_api_port }}/storage_service/tokens/{{ item }}"
+        method: GET
+      register: _existent_tokens
```
Comment (vladzcloudius) [inline ansible-scylla-node/tasks/generate_tokens.yml:38]: same here

File: ansible-scylla-node/tasks/generate_tokens.yml [hunk 4]
```diff
@@ -16,38 +16,48 @@
   loop: "{{ wait_for_cql_port_output.results }}"
   when: bootstrapped_node is not defined and item.failed == False
 
-- name: Validate that all nodes are in UN state and save token lists
```
Comment (vladzcloudius) [inline ansible-scylla-node/tasks/generate_tokens.yml:19]: Please, move it into a separate patch.

**Review by vladzcloudius — CHANGES_REQUESTED** [review]
Looks good in general. There are some comments though.

File: ansible-scylla-node/tasks/generate_tokens.yml (hunk 1 above)
Comment (igorribeiroduarte) [reply ansible-scylla-node/tasks/generate_tokens.yml:30]: Done

File: ansible-scylla-node/tasks/generate_tokens.yml (hunk 2 above)
Comment (igorribeiroduarte) [reply ansible-scylla-node/tasks/generate_tokens.yml]: Fixed

File: ansible-scylla-node/tasks/generate_tokens.yml (hunk 3 above)
Comment (igorribeiroduarte) [reply ansible-scylla-node/tasks/generate_tokens.yml]: Fixed

File: ansible-scylla-node/tasks/generate_tokens.yml (hunk 4 above)
Comment (igorribeiroduarte) [reply ansible-scylla-node/tasks/generate_tokens.yml:19]: Done
